src/main/java/com/nftworlds/wallet/objects/Wallet.java

```diff
@@ -78,6 +78,12 @@ public Wallet(UUID associatedPlayer, String address) {
         this.polygonWRLDBalance = polygonBalance;
         this.ethereumWRLDBalance = ethereumBalance;
 
+        NFTWorlds.getInstance().addWallet(this);
+    }
+
+    @Deprecated
+    public UUID getAssociatedPlayer() {
```

Comment:
There are only two references to `getAssociatedPlayer` which can both be replaced with `getOwner().getUuid()`. Could you change these to avoid adding a new and immediately deprecated method?

Reply:
Done my captain!

Reply:
I just didn't remove the getAssociatedPlayer() method so as not to break compatibility with projects of other developers who might use this method in their own projects.

Reply:
@iamarkdev @Jaxkr @Jaxkr And I have another question that is not related to the topic of this RP. I also found one small memory leak, in a couple of days I will try to make a new RP with a fix for this.

I want to design a better API package that is developer-friendly and better documented, but such significant changes would break compatibility with other developers' current projects. Would you accept such significant changes to the project? I just don't want to do a lot of work if it ends up getting rejected.

Now it's just that the API is designed very poorly, there is no separation between safe and unsafe, javadocs are very poorly documented, etc.

It's just that at the moment the project does not comply with the "Google Java Style Guide", "Java Code Conventions". There are no well-documented methods, new changes made can destroy compatibility with plugins of other developers, the developer has to interact with all the packages of the project, when it is more correct to make 1 common safe API package

As a developer, I need more events and callbacks (for example, if the transfer was canceled for some reason or if the player's balance changed) to track changes and better interact with your API. I have been working with him a lot in practice and he really needs to be improved, I would like to help with this.

![Screenshot_1](https://user-images.githubusercontent.com/35406568/166168781-4938ad06-4f07-42cc-9057-9c61fd60a109.png)
![Screenshot_2](https://user-images.githubusercontent.com/35406568/166168782-7e5862d5-6595-4502-81b4-2a7669411b9d.png)

Reply:
Hey, I would love to work with you directly on this. Please add me on Discord: Jax#4739

We’d want to minimize breakage, but we want the official NFT Worlds payment API to eventually offer a best-in-class developer experience. We learn more about what you want and we can work together to make it happen. 

Thanks so much for your PR and enthusiasm. Looking forward to hopefully collaborating with you.